braindecode/models/brainmodule.py

```diff
@@ -0,0 +1,729 @@
+# Authors: Meta Platforms, Inc. and affiliates (original)
+#          Bruno Aristimunha <b.aristimunha@gmail.com> (Braindecode adaptation)
+#          Hubert Banville <hubertjb@meta.com> (Braindecode adaptation and Review)
+#
+# License: Attribution-NonCommercial 4.0 International
+
+"""BrainModule: Dilated Convolutional Encoder for EEG classification."""
```

Comment:
Since it's been used for classification and regression
```suggestion
"""BrainModule: Dilated Convolutional Encoder for EEG decoding."""
```